extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/config/DialectVersions.java

```diff
@@ -0,0 +1,47 @@
+package io.quarkus.hibernate.orm.runtime.config;
+
+import org.hibernate.dialect.DatabaseVersion;
+
+/**
+ * Constants exposing the default versions of various DBs in Quarkus.
+ * <p>
+ * If, one day, Hibernate ORM's defaults catch up with all our default versions,
+ * we could consider relying on ORM's defaults going forward.
+ * <p>
+ * For H2, we will probably have to keep a constant here forever,
+ * as H2 is embedded and thus Quarkus determines its default version through the BOM.
+ * See https://github.com/quarkusio/quarkus/issues/1886
+ *
+ * @author Emmanuel Bernard emmanuel@hibernate.org
+ */
+public final class DialectVersions {
```

Comment:
I think I would have called it `DefaultDialectVersions` to be extra clear but the presence of `toString(DatabaseVersion version)` is not really in line with this.

Maybe rename the constant with `MARIADB_DEFAULT` or something similar?

It might just be me being over cautious but I find the intent not as clear as it should be. Feel free to ignore though if you think otherwise.

Reply:
I'll wrap this in a static `Defaults` class :)